docs/operations/data-center/index.md

```diff
@@ -0,0 +1,72 @@
+---
+seo:
+  title: Analyze data center power usage 
+  description: This recipe analyzes telemetry data from data center power electrical smart panels. The stream processing use cases for this data include detection of power usage levels for safety and accounting purposes.
+---
+
+# Analyze data center power usage 
+
+Your business provides a cloud infrastructure offering. Multiple physical data center locations are partitioned into smaller tenants, occupied by your customers. These individual tenants allow customers to operate in isolation and provide you with an accounting unit to monitor and accurately invoice your customers.
+
+The data centers consume large amounts of electricity which needs to be monitored to ensure business continuation. Additionally, the cost of the electricity needs to be accounted for and billed to the appropriate customer.
+
+Each data center is constructed with smart electrical panels that control the power supplies to multiple customer tenants. The smart panels emit telemetry data that is captured and produced into an Apache Kafka® topic. 
+
+![](diagram.svg)
+
+How can we utilize [ksqlDB](https://ksqldb.io/) to process the control panel telemetry data in real time?
+
+## Step-by-step
+
+### Setup your Environment
+
+--8<-- "docs/shared/ccloud_setup.md"
+
+### Read the data in
+
+Our data center power analysis applications require data from two different sources: customer tenant information and smart control panel readings.
+
+Typically, customer information would be sourced from an existing database. As customer occupancy changes, tables in the database would be updated and we can capture those changes and stream them into Kafka with Kafka Connect using [Change Data Capture](https://www.confluent.io/blog/cdc-and-streaming-analytics-using-debezium-kafka/).
+
+Telemetry data may be sourced into Kafka in a variety of ways. MQTT is a popular source for Internet of Things (IoT) devices, and smart electrical panels may provide this functionality out of the box. If the panel data can be sourced from MQTT, an [MQTT Connector](https://docs.confluent.io/kafka-connect-mqtt/current/mqtt-source-connector/index.html) is available to bridge MQTT and Kafka.
+
+Below are sample Kafka Connect configurations you could use to deploy source connectors to read data from their originating systems. 
+
+```sql
+--8<-- "docs/operations/data-center/source.json"
+```
+
+The above `MySqlCdcSource` configuration could be used to stream changes from the `customer` database's `tenant` table into the Kafka cluster. This connector is provided as [fully managed by Confluent Cloud](https://docs.confluent.io/cloud/current/connectors/cc-mysql-source-cdc-debezium.html).  And the same is true for the telemtry data, which can be sourced using the [fully managed MQTT source Connector](https://docs.confluent.io/cloud/current/connectors/cc-mqtt-source.html) on Confluent Cloud.
```

Comment:
MQTT is already mentioned above.  Can these paragraphs be merged to avoid the duplication and shorten the description?

Reply:
@ybyzek please see latest on this